### ISSUE TITLE
Add some "content tracking" to demo

### ISSUE THREAD

**Issue by hpvd** [matomo-org/matomo#10055]
Add some content tracking ("content impression" and "content-interaction") to demo, on https://demo.piwik.org/
**to demonstrate how content tracking reports look like**
and prevent empty sites.

One simple idea would be the tracking of "Piwik.org Blog widget" in dashboard
see:
![2016-04-14_15h47_32](https://cloud.githubusercontent.com/assets/5681880/14530211/36b03628-0258-11e6-8c4b-6768b404befd.png)

advantages choosing this element for demo:
- it's not always visible (only after scrolling on most screens -> impression <100%)
- it has some outgoing links inside like a typical use case of content tracking the "advertisment tracking" would have too...

Otherwise it looks like this :-(

![2016-04-14_15h34_06](https://cloud.githubusercontent.com/assets/5681880/14529930/00c72ab8-0257-11e6-881b-bf857b2ff210.png)

**Comment by tsteur** [comment]
👍 would be nice to have on demo.piwik.org. Not sure if we can do this with the forum

btw: done here as well https://demo-anonymous.piwik.org/index.php?module=CoreHome&action=index&idSite=1&period=year&date=today#?module=Contents&action=index&idSite=1&period=year&date=today

**Comment by hpvd** [comment]
@tsteur: since piwik is not doing selftracking here, of course these has to be added to forum-website - sorry
Thanks for pointing to other demo!

**Comment by Findus23** [comment]
demo.matomo.org now has content tracking data:

https://demo.matomo.org/index.php?module=CoreHome&action=index&idSite=62&period=day&date=yesterday#?idSite=62&period=day&date=yesterday&category=General_Actions&subcategory=Contents_Contents
